ras_party/controllers/account_controller.py

```diff
@@ -69,6 +70,8 @@ def post_respondent(party, tran, session):
     if not iac.get('active'):
         raise ClientError("Enrolment code is not active", status=400)
 
+    disable_iac(party['enrolmentCode'])
```

Comment:
- The documentation for this `post_respondent` method needs to be updated now that we're no longer creating a case event and are disabling the enrolment code.
- If there is a failure when committing the database records does that mean the respondent would be unable to retry (as the enrolment code would be disabled)?

Reply:
The behaviour is definitely different now. I followed these steps:
- Attempt to create an account with an existing email address
- On the validation error for the duplicate email address, change it to one that doesn't already exist and submit

On concourse-latest it works, on this branch a 500 error is generated.